std/format/package.d

```diff
@@ -269,119 +269,242 @@ It's also possible to use positional arguments for $(I width), $(I
+            $(TD Natural notation: Integral digits followed by a dot and
+                 fractional digits.
+
+                 When there are no fractional digits and the $(B '#') flag
+                 is $(I not) present, the dot is omitted.
+
+                 $(I Please note: the difference between $(B 'f') and $(B 'F')
+                 is only visible for `NaN` and `Infinity`.))
+   )
+   $(TR $(TD $(B 's'), $(B 'g'), $(B 'G'))
+            $(TD Short notation: If the absolute value is larger as `10 ^^ precision`
+                 or smaller than `0.0001`, the scientific notation is used.
+                 If not, the natural notation is applied.
+
+                 In both cases $(I precision) denotes the count of all digits, including
+                 the integral digits. Trailing zeros (including a trailing dot) are removed.
+
+                 If $(B '#') flag is present, trailing zeros are not removed.)
+   )
+   $(TR $(TD $(B 'a'), $(B 'A'))
+            $(TD Hexadecimal scientific notation: `0x` followed by `1`
+                 (or `0` in case of value zero or denormalized number)
+                 followed by a dot, fractional digits in hexadecimal
+                 notation and an exponent. The exponent is build by `p`,
+                 followed by a sign and the exponent in $(I decimal) notation.
+
+                 When there are no fractional digits and the $(B '#') flag
+                 is $(I not) present, the dot is omitted.)
+   )
+   $(TR $(TD $(B 'r'))
+            $(TD Characters taken directly from the binary representation.)
+   )
+   $(TR $(MULTIROW_CELL 3, $(I Character))
+        $(TD $(B 's'), $(B 'c'))
+            $(TD Like the character.
+
+                 Inside of a compound indicator $(B 's') is treated differently: The
+                 character is surrounded by single quotes and non printable
+                 characters are escaped. This can be avoided by preceding
+                 the compound indicator with a $(B '-') flag
+                 $(LPAREN)e.g. `"%-$(LPAREN)%s%$(RPAREN)"`$(RPAREN).)
+   )
+   $(TR $(TD $(B 'b'), $(B 'd'), $(B 'o'), $(B 'u'), $(B 'x'), $(B 'X'))
+            $(TD Like the integral that represents the character.)
+   )
+   $(TR $(TD $(B 'r'))
+            $(TD Characters taken directly from the binary representation.)
+   )
+   $(TR $(MULTIROW_CELL 3, $(I String))
+        $(TD $(B 's'))
+            $(TD The sequence of characters that form the string.
+
+                 Inside of a compound indicator the string is surrounded by double quotes
+                 and non printable characters are escaped. This can be avoided
+                 by preceding the compound indicator with a $(B '-') flag
+                 $(LPAREN)e.g. `"%-$(LPAREN)%s%$(RPAREN)"`$(RPAREN).)
+   )
+   $(TR $(TD $(B 'r'))
+            $(TD The sequence of characters, each formatted with $(B 'r').)
+   )
+   $(TR $(TD compound)
+            $(TD Like an array of characters.)
```

Comment:
Doesn't "As an array of characters" sound better than "Like an array of characters"? I, personally, don't like the latter. If you agree, please replace all such occurrences with "as".

Reply:
I was quite unsure here. The German translation is the same for both. I'll change everything to "as".